fs/README.md

```diff
@@ -0,0 +1,160 @@
+
+## Usage
+
+All the following modules are exposed in `mod.ts`
+
+### empty_dir
+
+Ensures that a directory is empty. Deletes directory contents if the directory is not empty.
+If the directory does not exist, it is created.
+The directory itself is not deleted.
+
+```ts
+import { emptyDir, emptyDirSync } from "https://deno.land/std/fs/mod.ts";
+
+emptyDir("./foo"); // returns a promise
+emptyDirSync("./foo"); // void
+```
+
+### ensure_dir
+
+Ensures that the directory exists.
+If the directory structure does not exist, it is created. Like mkdir -p.
+
+```ts
+import { ensureDir, ensureDirSync } from "https://deno.land/std/fs/mod.ts";
+
+ensureDir("./bar"); // returns a promise
+ensureDir("./ensureDirSync"); // void
+```
+
+### ensure_file
+
+Ensures that the file exists.
+If the file that is requested to be created is in directories
+that do not exist, these directories are created.
+If the file already exists, it is **NOT MODIFIED**.
+
+```ts
+import { ensureFile, ensureFileSync } from "https://deno.land/std/fs/mod.ts";
+
+ensureFile("./folder/targetFile.dat"); // returns promise
+ensureFileSync("./folder/targetFile.dat"); // void
+```
+
+### eol
+
+Detects and format the passed string for the targeted End Of Line character.
+
+```ts
+import { format, detect, EOL } from "https://deno.land/std/fs/mod.ts";
+
+const CRLFinput = "deno\r\nis not\r\nnode";
+const Mixedinput = "deno\nis not\r\nnode";
+const LFinput = "deno\nis not\nnode";
+const NoNLinput = "deno is not node";
+
+detect(LFinput); // output EOL.LF
+detect(CRLFinput); // output EOL.CRLF
+detect(Mixedinput); // output EOL.CRLF
+detect(NoNLinput); // output null
+
+format(CRLFinput, EOL.LF); // output "deno\nis not\nnode"
+...
+```
+
+### exists
+
+Test whether or not the given path exists by checking with the file system
+
+```ts
+import { exists, existsSync } from "https://deno.land/std/fs/mod.ts";
+
+exists("./foo"); // returns a Promise<boolean>
+existsSync("./foo"); // returns boolean
+```
+
+### glob
+
+Generate a regex based on glob pattern and options
+This was meant to be using the the `fs.walk` function
+but can be used anywhere else.
+
+```ts
+import { glob } from "https://deno.land/std/fs/mod.ts";
+
+glob("foo/**/*.json", {
+  flags: "g",
+  extended: true,
+  globstar: true
+}); // returns the regex to find all .json files in the folder foo
+```
+
+### move
+
+Moves a file or directory. Overwrites it if option provided
+
+```ts
+import { move, moveSync } from "https://deno.land/std/fs/mod.ts";
+
+move("./foo", "./bar"); // returns a promise
+moveSync("./foo", "./bar"); // void
+moveSync("./foo", "./existingFolder", { overwrite: true });
+// Will overwrite existingFolder
+```
+
+### read_json
+
+Reads a JSON file and then parses it into an object
+
+```ts
+import { readJson, readJsonSync } from "https://deno.land/std/fs/mod.ts";
+
+const f = await readJson("./foo.json");
+const foo = readJsonSync("./foo.json");
+```
+
+### utils
```

Comment:
I think `utils` modules should not expose by the user.

currently only used in internal modules, may change d in the future.

Reply:
Right it's not exposed in mod.ts